src/preact/contextprops.js

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-import {Loading, reducer as loadingReducer} from '../loading';
+import {Loading, reducer as loadingReducer} from './loading-instructions';
```

Comment:
This removes the only restricted dependency, so this file was removed from the allowlist

Reply:
Can we instead whitelist `src/context` and `src/contextprops`? Or move them to `src/pure`? They have been intentionally constructed as "pure". There might be some dependencies there now that have crept in, but I do think we can and should be "pure".